tests/cache_sorted_set.rs

```diff
@@ -59,37 +65,76 @@ async fn sorted_set_put_element_nonexistent_cache() {
     assert!(matches!(result.to_string(), _err_message))
 }
 
+fn compare_by_first_entry(a: &(String, f64), b: &(String, f64)) -> std::cmp::Ordering {
+    a.0.cmp(&b.0)
+}
+
 #[tokio::test]
 async fn sorted_set_put_elements_happy_path() {
+    async fn test_put_elements_happy_path(
+        client: Arc<CacheClient>,
+        cache_name: String,
+        to_put: impl IntoSortedSetElements<String> + Clone,
+    ) {
+        let sorted_set_name = "sorted-set-".to_string() + &Uuid::new_v4().to_string();
+        let result = client
+            .sorted_set_fetch_by_score(cache_name.clone(), sorted_set_name.clone(), Ascending)
```

Comment:
is it possible to avoid cloning cache_name and sorted_set_name in the tests?

Reply:
The way we've written the cache client, the methods take ownership of all the arguments. So when we want to re-use the cache name (sorted set name), we have to clone first.

Looking at the client, it looks like this is the case for `cache_name` (`impl Into<String>`) and the collection names (`impl IntoBytes`).

@cprice404 have we discussed elsewhere about what convention we want for variables that will get passed around a lot like the cache name or collection name? Seems we would want a reference for those to not waste effort copying many times.

Reply:
+1

Reply:
can you make your test's cache_name argument be `impl Into<String>`?

Can you make your sorted_set_name variable be `&str`?

Reply:
I was able to remove the clones on the cache name by passing references in [1b1d25c](https://github.com/momentohq/client-sdk-rust/pull/207/commits/1b1d25ca55f28967395bb1847c9df86fdb9f81c5).

For the sorted set name, I would have to create two variables, one with the `String` we build, then another that is a string reference. The alternative is in [722f53a](https://github.com/momentohq/client-sdk-rust/pull/207/commits/722f53a71fbb510d78cf042f23101eb021ffd58d) which calls `as_str` on each invocation.